github_conf/branch_protection_rules.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Why is this file here?

Reply:
I think this was resolved in Slack. It... was auto-generated? I know I did not explicitly create it. The suspect is `megalinter`.